src/test_graph.py

```diff
@@ -3,6 +3,52 @@
 
 
 @pytest.fixture
+def traverse():
+    """."""
+    from graph import Graph
+    trans_graph = Graph()
```

Comment:
Consider using loops to add edges or enable your graph to handle iterables.  This will clean up your code.